internal/config/cgmgr/systemd_linux.go

```diff
@@ -192,10 +208,20 @@ func (m *SystemdManager) MoveConmonToCgroup(cid, cgroupParent, conmonCgroup stri
 		}
 
 		if resources.CPU.Shares != nil {
-			props = append(props, systemdDbus.Property{
-				Name:  "CPUShares",
-				Value: dbus.MakeVariant(resources.CPU.Shares),
-			})
+			if node.CgroupIsV2() {
```

Comment:
I'm curious why the IPPR test fails when conmon cgroup doesn't have cpu weight set. it's checking the pod cgroup weight and that's the culmination of its child cgroups?